package.json

```diff
@@ -28,12 +28,12 @@
   },
   "license": "ISC",
   "peerDependencies": {
-    "babel-core": "6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc"
+    "@babel/core": "7 || ^7.0.0-beta || ^7.0.0-rc"
   },
   "devDependencies": {
-    "babel-core": "7.0.0-beta.0",
-    "babel-polyfill": "7.0.0-beta.0",
-    "babel-preset-env": "2.0.0-beta.0",
+    "@babel/core": "7.0.0-beta.5",
```

Comment:
It is already at 36 :)

Reply:
And in the meantime beta 37 was released. Gotta go fast! 😄